src/main/java/CarGame2/domain/model/Car.java

```diff
@@ -0,0 +1,38 @@
+package CarGame2.domain.model;
+
+public class Car {
+
+    private static final int MIN_SPEED = 4;
+    private static final int MAX_NAME_LENGTH = 5;
+    private final String name;
+
+    private int position = 0;
+
+    public Car(String name) {
+        validateName(name);
+        this.name = name;
+    }
+
+    private void validateName(String name) {
+        if (name == null || name.trim().isEmpty()) {
```

Comment:
```suggestion
        if (name == null || name.isBlank()) {
```

`isBlank()` 를 사용할 수도 있어요.

그리고 null 검사를 먼저 해주는 디테일이 정말 좋았어요. 이유를 알고 먼저 검사해주셨던 걸까요?

Reply:
사실.. 크게 고민하지 않고 작성했던 코드였습니다..ㅜ
리뷰어님 코멘트를 보고 이에 대해 찾아봤는데
null인 상태에서 isBlank()를 호출하면 NullPointerException이 발생하기에 null인지 검사를 먼저 해주어야 한다는 글을 보았습니다!  
리뷰어님도 이러한 이유로 말씀해주신 게 맞을까요..?

Reply:
정확합니다~ 👍 
NPE 방지는 코드 설계 시에 정말 중요한 요소라서 언급해봤어요.